src/main/java/seedu/duke/commands/WithdrawCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.duke.commands;
+
+import seedu.duke.AccountList;
+import seedu.duke.Currency;
+import seedu.duke.constants.ErrorMessage;
+import seedu.duke.constants.Message;
+import seedu.duke.exceptions.InsufficientAccountBalance;
+import seedu.duke.exceptions.InvalidAddCommandException;
+import seedu.duke.exceptions.NegativeWithdrawalAmountException;
+import seedu.duke.exceptions.NoAccountException;
+import seedu.duke.ui.Ui;
+
+/**
+ * This class is used to deal with the withdrawCommand.
+ */
+public class WithdrawCommand extends Command {
+    private Currency currency;
+    private int amount;
+
+    /**
+     * @param input   The user input including the command.
+     */
+    public WithdrawCommand(String input) {
+        super(false, input);
+    }
+
+    private Currency getCurrency(String currencyString) {
+        return Currency.valueOf(currencyString);
+    }
+
+    private void processCommand() throws InvalidAddCommandException, NegativeWithdrawalAmountException {
+        String[] words = super.input.split(" ");
+        // Format: [Command, CURRENCY, AMOUNT]
+        boolean isValidCommand = words.length == 3;
+        if (!isValidCommand) {
+            throw new InvalidAddCommandException();
+        }
+        this.currency = getCurrency(words[2]);
+        this.amount = Integer.parseInt(words[1]) * 100;
```

Comment:
I think we are going to read in as a float prior to multiplying by 100, so it should be 
```suggestion
        this.amount = Float.parseFloat(words[1]) * 100;
```